pkgs/development/tools/misc/coccinelle/default.nix

```diff
@@ -1,21 +1,36 @@
-{ fetchurl, lib, stdenv, python3, ncurses, ocamlPackages, pkg-config }:
+{ fetchFromGitHub, lib, stdenv, python3, ncurses, ocamlPackages, pkg-config, autoconf, automake }:
 
 stdenv.mkDerivation rec {
   pname = "coccinelle";
-  version = "1.1.0";
+  version = "1.1.1";
 
-  src = fetchurl {
-    url = "https://coccinelle.gitlabpages.inria.fr/website/distrib/${pname}-${version}.tar.gz";
-    sha256 = "0k0x4qnxzj8fymkp6y9irggcah070hj7hxq8l6ddj8ccpmjbhnsb";
+  src = fetchFromGitHub {
```

Comment:
Why are you switching to GitHub? If there are official release tarballs, they should be used instead, as they are specifically intended for packaging purpose (whereas GitHub repos are primarily development-oriented).

The release tarballs also have the configure scripts already generated, so you no longer need `automake`, `autoconf` etc.

Reply:
Well, the official page is pointing to https://coccinelle.gitlabpages.inria.fr/website/distrib/coccinelle-1.1.1.tar.gz
which is 404.

Reply:
Can you add a comment about this in the source? we should definitely switch back in case it becomes available again.

Reply:
Added

Reply:
Did you report this dead link issue upstream?

Reply:
coccinelle/coccinelle#279